build/Repositories.props

```diff
@@ -1,55 +1,55 @@
 <Project>
-  <ItemDefinitionGroup>
-    <Repository>
-      <Branch>patch/2.0.1</Branch>
```

Comment:
FYI - I want to see if we can create our patch system in such a way that we don't have to create and build these patch branches.